lib/internal/Magento/Framework/Image/Adapter/Gd2.php

```diff
@@ -63,7 +61,7 @@ protected function _reset()
      */
     public function open($filename)
     {
-        if (!file_exists($filename)) {
+        if (!$filename || !file_exists($filename)) {
             throw new FileSystemException(
                 new Phrase('File "%1" does not exist.', [$this->_fileName])
```

Comment:
Hello @anzin 
Perhaps `$_fileName` property hasn't set yet.
Could you please check my suggestion

```suggestion
                new Phrase('File "%1" does not exist.', [$filename])
```

Reply:
Code fixed by recommendation, please check if every think is well.

Thank you.